test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -61,9 +61,8 @@ def run_smdebug_test(
     try:
         test_output = ec2_connection.run(
             f"{docker_executable} exec --user root {container_name} "
-            f"/bin/bash -c '{test_script} {framework}' | tee {logfile}",
+            f"/bin/bash -c '{test_script} {framework}' >> {logfile}",
             hide=True,
-            warn=True,
```

Comment:
Why do we remove this flag?

Reply:
This is set warn=True so that we can raise the actual fabric error along with additional context

```
        raise SMDebugTestFailure(
            f"SMDebug test failed on {image_uri} on {ec2_instance_type}. Full output:\n{debug_stdout}"
        ) from e
```

Now we will raise more than just assertion error if the test fails - we will have the fabric error referenced in the "**from e**" and the full debug information from docker logs in debug_stdout